src/data_warehouse/core/__init__.py

```
@@ -0,0 +1,12 @@
+"""Core functionality for the data warehouse."""
+
+from data_warehouse.core.exceptions import (
+    ApiError,
```

Ruff (0.8.2)

`data_warehouse.core.exceptions.ApiError` imported but unused; consider removing, adding to `__all__`, or using a redundant alias

(F401)